### PR TITLE
Implement Blogging Prompts introduction dialog boilerplate

### LINKED ISSUE #16109: [Blogging prompts v1] Introduction - Dialog Scaffolding and UI
Implement blogging prompts feature onboarding dialog UI.

### PR DESCRIPTION
**Pull request by RenanLukas**
Fixes #16109 (PR 1/2)

To test:
- This is just boilerplate of the onboarding dialog. It's not wired to any screen and doesn't actually implement a feature, so it can't be tested right now.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Nothing yet.

3. What automated tests I added (or what prevented me from doing so)
None yet.

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/onboarding/OnboardingDialogFragment.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import androidx.lifecycle.ViewModelProvider
+import org.wordpress.android.R
+import org.wordpress.android.R.attr
+import org.wordpress.android.WordPress
+import org.wordpress.android.databinding.BloggingPromptsOnboardingDialogFragmentBinding
+import org.wordpress.android.util.getColorFromAttribute
+import org.wordpress.android.util.isDarkTheme
+import javax.inject.Inject
+
+class OnboardingDialogFragment : DialogFragment() {
```
Comment (khaykov) [inline WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/onboarding/OnboardingDialogFragment.kt]: This is very minor, but what do you think about naming this `BloggingPromptsOnboardingDialogFragment` just to be a bit more specific?

File: WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/onboarding/OnboardingViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package org.wordpress.android.ui.bloggingprompts.onboarding
+
+import androidx.lifecycle.ViewModel
+
+class OnboardingViewModel : ViewModel()
```
Comment (khaykov) [inline WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/onboarding/OnboardingViewModel.kt]: Same naming related comment here as well (`BloggingPromptsOnboardingViewModel`).

**Review by khaykov — APPROVED** [review]
Looks good, @RenanLukas ! Just have a very minor naming related comment :)

**Comment by RenanLukas** [comment]
Thanks for the review @khaykov 👍 
I've implemented your suggestions

**Review by khaykov — APPROVED** [review]
Thanks for the changes! Looks good 👍

**Pull request merged**